mlos_bench/mlos_bench/optimizers/one_shot_optimizer.py

```diff
@@ -18,20 +18,27 @@
 
 class OneShotOptimizer(MockOptimizer):
     """
-    Null optimizer that proposes a single configuration and converges.
+    Mock optimizer that proposes a single configuration and returns.
+    Explicit configs (partial or full) are possible using configuration files.
     """
 
+    # TODO: Add support for multiple explicit configs (i.e., FewShot or Manual Optimizer).
+
     def __init__(self, tunables: TunableGroups,
                  service: Optional[Service], config: Dict[str, Any]):
         super().__init__(tunables, service, config)
         if self._service is None:
             _LOG.warning("Config loading service not available")
         else:
-            for data_file in config.get("include_tunables", []):
+            # Generate initial random or default suggestion based on optimizer config.
+            self._tunables = super().suggest()
```

Comment:
Since the parent is MockOptimizer, this will call the appropriate default or random logic for us.
Then below we can overwrite those values provided from the config.

Reply:
makes sense!